ArrCharOps.java

```diff
@@ -37,6 +37,9 @@ public static void println(char[] arr) {
      */
     public static char charAt(char[] arr, int index) {
```

Comment:
Try and use a `.gitignore` file that fits your InteliJ IDE. 
a lot of this is not needed